README.md

```diff
@@ -0,0 +1,115 @@
+![Page Glide Logo](images/page_glide_logo.png)
+
+Building an online bookstore requires a strong foundation. 
+Page Glide was created to empower developers with a well-structured and efficient backend API 
+for their online bookstore applications.
+
+## Introduction
+Page Glide is a backend API built with Spring Boot for rapid development. 
+It leverages Spring Security for secure user authentication and authorization. 
+Spring Data JPA provides seamless interaction with a MySQL database for data persistence. 
+Swagger is integrated for clear and comprehensive API documentation. 
+Docker enables easy containerization for deployment.
+
+### Key features
+
+* User Authentication and Authorization (Registration, Login)
+* Book Management (CRUD operations, searching)
+* Category Management (CRUD operations)
+* Shopping Cart Management (adding, updating, removing items)
+* Order Management (placing orders, viewing order history, order details)
+* User Role Management (Admin vs. User functionalities)
+* Unit Testing (ensuring code reliability)
+
+## Technologies
+* **Spring Boot**: framework for building Java-based web applications
+* **Spring Security**: security and user management
+* **Spring Data JPA**: data persistence
+* **Swagger**: API documentation
+* **Docker**: deployment and scalability
+* **Liquibase**: database management
+* **MySQL**: database management system
+* **Mockito**: mocking frameworks for tests
+* **JUnit**: testing and reliability
+
+
+## Installation
+
+### 1. Prerequisites
+* Java 21+
+* Maven 4+
+* Docker
+* MySQL 8+
```

Comment:
we can mention that we need all that in case we want to boot the app using maven spring boot run, and if we are using docker - we do not need all that